axonserver/src/main/java/io/axoniq/axonserver/localstorage/transformation/EventTransformerFactory.java

```diff
@@ -21,8 +19,8 @@
      * Get an event transformer. The transformer may be based on version and flags (for existing data) or based on settings in the storage properties (for new data).
      * @param version the version for existing data
      * @param flags the flags for existing data
-     * @param storageProperties storage properties to consider for new data
+     * @param context the context for which to create an event transformer
      * @return the transformer
      */
-    EventTransformer get(byte version, int flags, StorageProperties storageProperties);
+    EventTransformer get(byte version, int flags, String context);
```

Comment:
I think that the context is need only for one implementation and should be added as a field of that implementation i.o. as a param here

Reply:
I agree. The issue was that there was only one factory for all contexts. Updated EE code to handle multi-context, making the context here no longer needed.